requirements/dev.txt

```diff
@@ -5,7 +5,6 @@ pre-commit==1.15.1
 tox==3.1.3
 django-debug-panel==0.8.3
 django-debug-toolbar==1.9.1
-ruamel.yaml.clib==0.2.2
```

Comment:
If this specific requirement is _only_ required for Python 2.7, then we could instead codify that like this:
```
ruamel.yaml.clib==0.2.2; python_version < "3"
```

This should restrict the installation only to Python 2.7